src/distribution/binomial.rs

```diff
@@ -242,9 +242,7 @@ impl distribution::Median for Binomial {
         use distribution::Inverse;
         use std::f64::consts::LN_2;
 
-        if self.np.fract() == 0.0 {
-            self.np
-        } else if self.p == 0.5 && self.n % 2 != 0 {
+        if self.np.fract() == 0.0 || (self.p == 0.5 && self.n % 2 != 0) {
```

Comment:
Why is this change needed?

Reply:
I don't think it's necessary, but `rustc` raised a warning that both the `if` and the first `else if` branch had identical bodies, so they could be collapsed into a single branch.

Reply:
I guess the warning is mostly meant to catch copy+paste mistakes where the bodies should actually be different.

Reply:
I see. Then it is justified.